src/ClientBuilder.php

```diff
@@ -219,6 +224,18 @@ public function setRepresentationSerializer(RepresentationSerializerInterface $r
         return $this;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setSdkIdentifierSuffix(?string $sdkIdentifierSuffix): void
```

Comment:
I don't think we should expose a method to set just the suffix on the SDK identifier. I would prefer to be able to set the whole name (there is no server-side validation regarding this value).